File: src/ClientApp/src/app/pages/clients/view-models/client-updated.view-model.ts
export interface ClientUpdatedViewModel {
    id: string;
    clientCode: string;
    name: string;
    countryName: string;
    industryName?: string | null;
    status: number;
    statusName: string;
    accountOwnerUserName?: string | null;
    updatedAt?: string | null;
}

File: src/API/src/Modules/Clients/Clients.Application/ViewModels/ClientUpdatedViewModel.cs
using Clients.Domain.Enums;

namespace Clients.Application.ViewModels
{
    public class ClientUpdatedViewModel
    {
        public Guid Id { get; set; }
        public string ClientCode { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string CountryName { get; set; } = string.Empty;
        public string? IndustryName { get; set; }
        public ClientStatus Status { get; set; }
        public string StatusName { get; set; } = string.Empty;
        public string? AccountOwnerUserName { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}
